lab_4_summarization_textrank/main.py

```diff
@@ -304,7 +466,14 @@ def _find_texts_close_to_keywords(self, keywords: tuple[str, ...], n_texts: int)
         :param n_texts: number of texts to find
         :return: the texts' ids
```

Comment:
train of what happened during demonstration

Reply:
of TextRankSummarizer